packages/core/src/babel-plugin-adjust-imports.ts

```diff
@@ -255,7 +270,7 @@ function makeHBSExplicit(specifier: string, sourceFile: AdjustFile) {
   return specifier;
 }
 
-export default function main({ types: t }: { types: any }) {
```

Comment:
I thought the convention was to use the `t` provided during usage of the plugin rather then via what is imported

Reply:
Ultimately, I'm fine with it either way.

Reply:
Yeah, that is the traditional way I've seen it done. But typescript seems happier this way. It gives really good feedback on which properties are legal on which nodes and paths.

Like I said, this is mostly based off me reading other typescript plugins to see how they do things.